src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,612 @@
+                          doAnswer(invocation -> {
+                              return "mockMsg";
+                          }).when(mock).getMsg();
+                      });
+                  }
+              
+                  @After
+                  public void tearDown() {
+                      mockObjMockUpTest_MyClazz.closeOnDemand();
+                  }
+              
+                  @Test
+                  public void test() {
+                      assertEquals("mockMsg", new MyClazz().getMsg());
+                  }
+              
+                  public static class MyClazz {
+                      public String getMsg() {
+                          return "msg";
+                      }
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    public void mockUpAtSetUpWithTearDownTest() {
+        rewriteRun(
+          java(
+            """
+              import org.junit.Before;
+              import org.junit.After;
+              import org.junit.Test;
+              import mockit.Mock;
+              import mockit.MockUp;
+              import static org.junit.Assert.assertEquals;
+              
+              public class MockUpTest {
+                  @Before
+                  public void setUp() {
+                      new MockUp<MyClazz>() {
+                          @Mock
+                          public String getMsg() {
+                              return "mockMsg";
+                          }
+              
+                          @Mock
+                          public String getStaticMsg() {
+                              return "mockStaticMsg";
+                          }
+                      };
+                  }
+              
+                  @After
+                  public void tearDown() {
+                  }
+              
+                  @Test
+                  public void test() {
+                      assertEquals("mockMsg", new MyClazz().getMsg());
+                      assertEquals("mockStaticMsg", MyClazz.getStaticMsg());
+                  }
+              
+                  public static class MyClazz {
+                      public String getMsg() {
+                          return "msg";
+                      }
+              
+                      public static String getStaticMsg() {
+                          return "staticMsg";
+                      }
+                  }
+              }
+              """,
+            """
+              import org.junit.Before;
+              import org.junit.After;
+              import org.junit.Test;
+              import org.mockito.MockedConstruction;
+              import org.mockito.MockedStatic;
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.Mockito.*;
+              
+              public class MockUpTest {
+                  private MockedConstruction mockObjMockUpTest_MyClazz;
```

Comment:
Looks good. Best to change field name to standard Java naming convention of camel case. Also could add support for multiple mocked constructions/ mockedStatics and create lists here.

Reply:
This naming follows the convention of PowerMockitoMockStaticToMockito. I suppose we should align with it.

I don't really understand "support for multiple mocked constructions/ mockedStatics and create lists here.". You mean I should add more mock classes and fields for testing?

Reply:
lower camel case is preferred here
https://www.thoughtco.com/using-java-naming-conventions-2034199
https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

I'd change the variable name as well, it sounds very confusing. Probably something like mockedConstruction should be enough. No need to call it mockup, as migrating away from jmockit.

I mean add support via lists or avoid migration for following:

```
@Before
void setup() {
    new Mockup<Foo>() {
        // mocked methods
    }

    new Mockup<Bar>() {
          // mocked methods
    }
}
```

Would prefer if running your recipe doesn't break above code. Could you add test case(s) for the above scenario?